### PR TITLE
Changes alt highlight to have border and background

### LINKED ISSUE #43: Color tweaks
* Need to prune mentions of old color classes.
* Need to add #EBF3F3 to the editor palette (`highlight-alt-background`)?

### PR DESCRIPTION
**Pull request by ChaosExAnima**
Fixes #43.

Makes it so there are a `highlight-alt` text and border color, one for text and one for lighter applications like backgrounds and borders.

### CODE REVIEW

File: assets/variables.css [hunk 1]
```diff
@@ -7,7 +7,8 @@
 	--color-tertiary-text: #47133d;
 	--color-tertiary-border: #a17193;
 	--color-highlight: #f7cf56;
-	--color-highlight-alt: #a4cacb;
+	--color-highlight-alt-text: #a4cacb;
+	--color-highlight-alt-border: #ebf3f3;
```
Comment (jffng) [inline assets/variables.css:11]: #43 suggests that this should be added to the editor palette. Is this PR supposed to take care of that too?

Comment (ChaosExAnima) [reply assets/variables.css:11]: Yes, as the `highlight-alt` color was added in #22. What this is doing is setting it so the one color is lighter for backgrounds while text remains the same. I can't find a way to declare this via Gutenberg.

Comment (jffng) [reply assets/variables.css:11]: I see, yeah, the palette supplied by the theme is used for both the background and text color. Why not just add the color explicitly to the palette? 

How it's done in this PR, the editor and front-end won't match:

**Editor**
Screen Shot 2020-05-14 at 12 06 47 AM

**Front end**
Screen Shot 2020-05-14 at 12 06 52 AM

Comment (ChaosExAnima) [reply assets/variables.css:11]: @jffng - Good point, may need to throw an `important` on there.

Regarding not creating a new color in the palette, as per the design, this is for backgrounds only so I don't want to clutter things up.

Comment (ChaosExAnima) [reply assets/variables.css:11]: @jffng - Tried to replicate but couldn't. Maybe a cache?

Screen Shot 2020-05-14 at 10 00 57 PM

Comment (ChaosExAnima) [reply assets/variables.css:11]: Okay, in bb2ae3e I hacked Gutenberg to display the correct colors:
Screen Shot 2020-05-15 at 10 49 26 PM
Screen Shot 2020-05-15 at 10 49 32 PM

It's far from perfect but works for now imo.

Comment (jffng) [reply assets/variables.css:11]: The picker is showing the correct color, but how the blocks actually display is still being overridden by the inline styles:

Screen Shot 2020-05-18 at 9 34 22 AM

I think if you wanted to override the inline styles, there would need to be a rule like the following in the editor stylesheet:

```css
.has-hightlight-alt-background-color {
   background-color: var( --color-highlight-alt-border ) !important;
}
```

Screen Shot 2020-05-18 at 9 37 41 AM

Comment (jffng) [reply assets/variables.css:11]: I was using an outdated version of Gutenberg, this LGTM as of 8.1.0.

**Pull request merged**